### ISSUE TITLE
Disconnecting does not close pts on server

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dragonlinux/connectbot#203]
```
What steps will reproduce the problem?
1. connect to a host using connectbot, then disconnect
2. connect to the same host using another SSH client like PuTTY, get a
'tree' view of processes by running the command 'ps auxf'
3. observe that the virtual console used by connectbot was not closed when
connectbot disconnected

What is the expected output? What do you see instead?
Expect to see that the connection by connectbot was fully closed, see this
from ps (lines numbered by me):
1  /usr/sbin/sshd
2   \_ sshd: andy [priv]
3   |   \_ sshd: andy@pts/0
4   \_ sshd: andy [priv]
5       \_ sshd: andy@pts/0
6           \_ -bash
7              \_ ps auxf
lines 2 and 3 are the processes started by the connectbot connection, 4-7
are a PuTTY session. When the connectbot session is ended, processes on
lines 2-3 should be killed

What version of the product are you using (you can see this by using Menu
-> About in the Host List)?
1.5.5 (r409 2009.09.06)


What type of system are you trying to connect to?
Ubuntu 9.04 Server edition (2.6.28-15-generic)


If you are able to connect, what is the output of "echo $TERM", "uname -a",
and any other relevant information on the host?
uname -a: Linux pandora 2.6.28-15-generic #52-Ubuntu SMP Wed Sep 9 10:49:34
UTC 2009 i686 GNU/Linux
with connectbot: echo $TERM = "screen"
with PuTTY: echo $TERM = "xterm"

Please provide any additional information below.

```

Original issue reported on code.google.com by `andyjobs...@gmail.com` on 8 Oct 2009 at 6:52

**Comment by GoogleCodeExporter** [comment]
```

11:26 <douglaswth> hrm, I saw a pts/1 from my connectbot session, but it went 
away
pretty soon afterwords
11:27 <andyjobson85> hmm, maybe the server kills it after a while
<andyjobson85> what OS did you connect to?
11:29 <douglaswth> Linux 2.6.28-15-server Ubuntu Jaunty
11:30 <andyjobson85> i'll try connecting again and check after a while
<douglaswth> ssh sessions that don't close down correctly usually get killed
eventually anyway
11:31 <andyjobson85> yeah.. but I wonder why they dont get closed down 
correctly?
11:34 <andyjobson85> i noticed something... if you press the menu key, then
'disconnect' it will close correcctly
<andyjobson85> but if you type exit at the prompt it wont close correctly

```

Original comment by `andyjobs...@gmail.com` on 17 Oct 2009 at 10:36

**Comment by GoogleCodeExporter** [comment]
```
I couldn't reproduce this with the 1.6 branch.
```

Original comment by `kenny@the-b.org` on 17 Oct 2009 at 4:26

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in 1.6 as far as I can tell.
```

Original comment by `kenny@the-b.org` on 25 Oct 2009 at 5:05
- Changed state: **WontFix**
